vllm/attention/backends/utils.py

```diff
@@ -33,10 +33,9 @@ def is_block_tables_empty(block_tables: Union[None, Dict]):
     """
     if block_tables is None:
         return True
-    if isinstance(block_tables, dict) and all(
-            value is None for value in block_tables.values()):
-        return True
-    return False
+    return bool(
+        isinstance(block_tables, dict)
+        and all(value is None for value in block_tables.values()))
```

Comment:
```suggestion
    return (
        isinstance(block_tables, dict)
        and all(value is None for value in block_tables.values())
    )
```

Reply:
huh strange, since ruff format run this

Reply:
Which rule is this under?

Reply:
Any update on this?

Reply:
This is just `ruff format` afaict

Reply:
Currently, we should use `yapf` instead of `ruff` for formatting. So you can apply my suggestion unless `ruff check` raises an error.